core/trino-main/src/main/java/io/trino/sql/planner/LogicalPlanner.java

```diff
@@ -658,7 +658,8 @@ private Expression noTruncationCast(Expression expression, Type fromType, Type t
 
     private RelationPlan createDeletePlan(Analysis analysis, Delete node)
     {
-        DeleteNode deleteNode = new QueryPlanner(analysis, symbolAllocator, idAllocator, buildLambdaDeclarationToSymbolMap(analysis, symbolAllocator), metadata, Optional.empty(), session, ImmutableMap.of())
+        Map<NodeRef<LambdaArgumentDeclaration>, Symbol> lambdaDeclarationToSymbolMap = buildLambdaDeclarationToSymbolMap(analysis, symbolAllocator);
```

Comment:
These seem unnecessary. Moreover, the extra variable (with the overly verbose name) makes it harder to read the code. If the length of the line is an issue, it would more readable to format this as:

```java
DeleteNode deleteNode = new QueryPlanner(
        analysis, 
        symbolAllocator, 
        idAllocator, 
        buildLambdaDeclarationToSymbolMap(analysis, symbolAllocator), 
        metadata, 
        Optional.empty(), 
        session, 
        ImmutableMap.of())
```